src/ext/framework/mod.rs

```diff
@@ -258,6 +258,48 @@ impl Framework {
         self
     }
 
+    /// Aliases a command, allowing it to be used under a different name.
+    /// Note that default group is called "Ungrouped".
+    pub fn alias<S>(mut self, group: S, to: S, from: S) -> Self
```

Comment:
Can we make these arguments `&str` types? I'm unconvinced that I want them to be `Into<String>`; the `Copy` bound just makes the `Into<String>` trait bound useless anyway